spec/factories/course_lesson.rb

```diff
@@ -4,5 +4,11 @@
   factory :course_lesson do
     title { "Test Course lesson" }
     course_module { FactoryBot.create(:course_module) }
+
+    trait :with_lesson_part do
```

Comment:
If the tests were failing, does that mean that this is required? If so, should it be a trait or should it just happen anyway?

(sorry for commenting on a PR you haven't requested a review on yet lol)

Reply:
Tests were failing because in case a lesson doesn't have a part, I don't show any content. And the tests, well, they expect content 😂